chipy_org/apps/meetings/urls.py

```diff
@@ -23,19 +25,18 @@
     url(r'^(?P<pk>[0-9]*)/$',
         MeetingDetail.as_view(), name='meeting'),
 
+    # pass the meeting id here!!!
```

Comment:
I'm not sure what this comment is directing. Is this a TODO or is describing a behavior of the code?